blockchain/blockchain_test.go

```diff
@@ -833,8 +833,9 @@ func TestBlockchainInitialCandidate(t *testing.T) {
 	defer func() {
 		require.NoError(bc.Stop(context.Background()))
 	}()
-	// TODO: we will fix this test case by testing using lifeLongDelegatesProtocol to initialize the candidates
-	// candidate, err := sf.CandidatesByHeight(0)
+	candidate, err := sf.CandidatesByHeight(1)
```

Comment:
Check this: https://github.com/iotexproject/iotex-core/blob/master/server/itx/server.go#L283-L319

Like the one you fix in minicluster, you need to adjust the genesis config to enable lifeLongDelegatesProtocol